index.html

```diff
@@ -150,32 +153,51 @@ <h2 class= "abt">About<br> Myself</h2>
+            <ul>
+              <li class= "lang"><h3>Languages</h3></li>
+              <li><img src="icons/Icon-Chevron-Down-Larger.png" alt="#" /></li>
+            </ul>
+          </div>
+          <div class="tags">
+            <ul>
+              <li class= "tools"><img src="icons/JS-Ellipse1.png" alt="#"/>Javascript</li>
+              <li class= "tools"> <img src="icons/HTML-Ellipse1.png" alt="#"/> HTML</li>
+              <li class= "tools"> <img src="icons/CSS-Ellipse1.png" alt="#"/> CSS</li>
+            </ul>
+          </div>
+          <div class= "titles">
               <ul>
-                <li class= "lang"><h3>Languages</h3></li>
+                <li class= "lang"><h3>Frameworks</h3></li>
                 <li><img src="icons/Icon-Chevron-Down-Larger.png" alt="#" /></li>
               </ul>
-            </section>
+          </div>
+          <div class="skill">
+            <ul>
+              <li class= "lang"><h3>Skills</h3></li>
+              <li><img src="icons/Icon-Chevron-Down-Larger.png" alt="#" /></li>
+            </ul>
+          </div>
+        </section>
 
-            <div class="tags">
-              <ul>
-                <li class= "tools"><img src="icons/JS-Ellipse1.png" alt="#"/>Javascript</li>
-                <li class= "tools"> <img src="icons/HTML-Ellipse1.png" alt="#"/> HTML</li>
-                <li class= "tools"> <img src="icons/CSS-Ellipse1.png" alt="#"/> CSS</li>
-              </ul>
-            </div>
-            <div class= "titles">
-                <ul>
-                  <li class= "lang"><h3>Frameworks</h3></li>
-                  <li><img src="icons/Icon-Chevron-Down-Larger.png" alt="#" /></li>
-                </ul>
-            </div>
-            <div class="skill">
-              <ul>
-                <li class= "lang"><h3>Skills</h3></li>
-                <li><img src="icons/Icon-Chevron-Down-Larger.png" alt="#" /></li>
-              </ul>
+        <!-- Contact section -->
+        <section class="contact-container">
+            <div class="contact-me">
+              <h2 class="contact-title">Contact Me</h2>
+                <p class="contact-notice">
+                  If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+                </p>
+                <form class="contact-form" id="form" action="https://formspree.io/f/xyyvwrpj" method="post">
+                  <input type="text" name="name" placeholder="Full name" class="input-setup" maxlength="30" required>
+                  <input type="email" name="email" id="em" placeholder="Email address" class="input-setup" required>
+                  <textarea name="message" placeholder="Write your massage here" class="input-setup text-area" maxlength="500" required></textarea>
+                  <button type="submit" class="contact-button">Get In Touch</button>
+                </form>
+                <hr class="basement">
```

Comment:
[Optional] Don't you think the size of your contact form is very different from the one in the Figma template? I'd recommend increasing the size of your text and also increasing the height of your contact section. So that your contact section will look very similar to the one in Figma template.

![image](https://user-images.githubusercontent.com/34779928/192875507-3ed9ba30-8429-4483-9858-af4ffb5340b0.png)